.cirrus.yml

```diff
@@ -0,0 +1,38 @@
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus
+
+task:
+  name: Test on Graviton2 ($CC)
+  arm_container:
+    image: ghcr.io/ruby/ruby-ci-image:$CC
+    cpu: 4
```

Comment:
Could you change the cpu to `cpu: 8`? We can use maximum 16 core. So, I think 8 x 2 matrix jobs = 16 is the way to use the core effectively.

Reply:
Changed in 322c71550c5a9baf5bdee52205080cd4d1effac7